contents/scala/Scala.md

```diff
@@ -0,0 +1,284 @@
+<frontmatter>
+  title: Introduction to Scala
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Scala
+
+**Author(s): [Wang Chao](https://github.com/fzdy1914)**
+
+Reviewers: [Jiang Chunhui](https://github.com/Adoby7), [Wang Junming](https://github.com/junming403)
+
+**Table of Contents**
+
+* [Overview](#overview)
+  * [Characteristic of Scala](#characteristic-of-scala)
+    * [Statically Typed](#statically-typed)
+    * [Object Oriented](#object-oriented)
+    * [Functional](#functional)
+* [Why Scala](#why-scala)
+  * [Operator Overloading](#operator-overloading)
+  * [Mixed-In Trait](#mixed-in-trait)
+  * [Type Enrichment](#type-enrichment)
+  * [Pattern Matching](#pattern-matching)
+* [Getting Started](#getting-started)
+* [Where to go from here](#where-to-go-from-here)
+
+## Overview
```

Comment:
I suggest you simply quote Scala website rather than try to rephrase the same thing using different words. This way, you can remain independent as the claims about the language (e.g., _elegant_, _concise_ etc.) are coming from an official source:

Example:

# Scala

## Overview

>Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries. -- [Scala Website](https://www.scala-lang.org/)

Let's look at some characteristics of Scala mentioned in the above quote:

* **Scala supports _object-oriented_ programming**: Every value in Scala is an object, including functions and primitives. Classes are extended by *subclassing* and a flexible *mixin-based* mechanism to avoid the problems of multiple inheritance.
* **Scala supports _functional_ programming**: Every function in Scala is a value. Scala provides a lightweight syntax for defining [*anonymous functions*](https://docs.scala-lang.org/tour/basics.html#functions). It also support many other features of FP including [*higher-order functions*](https://docs.scala-lang.org/tour/higher-order-functions.html), [*nested functions*](https://docs.scala-lang.org/tour/nested-functions.html), [*currying*](https://docs.scala-lang.org/tour/multiple-parameter-lists.html), [*pattern matching*](https://docs.scala-lang.org/tour/pattern-matching.html), *immutability* and *lazy evaluation*.
* **Scala is _statically typed_**: It enforces type checking like verifying and enforcing the constraints of 
types at compile-time. 
* **Scala can work the Java and JavaScript runtimes**: ...

## Noteworthy Scala Features

### Operator overloading
...

## Getting Started
...

Reply:
@damithc Do you mean that some add-on characteristics are not worth mention here, for example, type inference, Traits?

Reply:
Yes, let's limit to those mentioned in the quote from the website. Traits, type inference etc. can be mentioned as _features_; they are not characteristic (i.e., not related to the overall nature of the language).